src/treemodel/delta/delta.js

```diff
@@ -118,6 +131,31 @@ export default class Delta {
 	}
 
 	/**
+	 * Custom toJSON method to make deltas serializable.
+	 *
+	 * @returns {Object} Clone of this delta with added class name.
+	 */
+	toJSON() {
+		let json = clone( this );
+
+		json.__class = this.constructor.className;
+
+		json.batch = null;
```

Comment:
This can be changed, but should be changed in other places like operation where we have:
```js
json.parent = this.parent ? this.parent.name : null;
```
So this is consistent with others (setting null). So above should be changed as:

```js
if( this.parent ) {
    json.parent = this.parent.name;
} else {
    delete json.parent;
}
```